spec/support/accessible_autocomplete_helper.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AccessibleAutocompleteHelper
+  def accessible_autocomplete_select(option, for_id:, value: nil)
+    # accessible autocomplete replaces a select element with an input
+    autocomplete_input = find("input##{for_id}")
+    autocomplete_input.fill_in(with: option)
+
+    # select the option
+    parent_node = autocomplete_input.find(:xpath, "..")
+    parent_node.first(".autocomplete__option").click
+
+    # wait until option is selected
+    within("##{for_id}-select", visible: false) do
+      if value
+        find(%{option[value="#{value}"]:checked}, visible: false)
+      else
+        find("option:checked", visible: false, text: option)
```

Comment:
Looks good - not sure why we need this else clause right now, though?

Reply:
Yeah - it isn't needed here. It was a bit of a pain to work out so put it in. I think we should use it in other places. I've pulled it out for now. Will try bring it back soon though before it slips my mind.